pkg/Library/GraphsNodes/CameraNode.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+export const CameraNode = {
```

Comment:
is this intended to replace the CameraNode declared in `NewMedia/` folder?

(same question about ImageNode and DisplayNode below)

Reply:
It's a new set of Nodes so that I can explore new spaces without breaking the old stuff.